### PR TITLE
#6264 avoid ignoring the caught exception

### PR DESCRIPTION
**Pull request by asolntsev**
## Impact

- [x] Bug fix (non-breaking change which fixes expected existing functionality)
- [ ] Enhancement/New feature (adds functionality without impacting existing logic)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 

Similar to https://github.com/liquibase/liquibase/pull/6286, but in many other places.

I found all places where IDEA shows warning "catch block may ignore exception", and fixed many of them.

### CODE REVIEW

**Comment by lbadam** [comment]
@asolntsev thanks so much for submitting this PR. We'll review it and get back to you soon.

File: liquibase-standard/src/main/java/liquibase/executor/jvm/StatementCreatorUtils.java [hunk 1]
```diff
@@ -80,7 +82,7 @@ private static void setParameterValueInternal(
                     useSetObject = (ps.getConnection().getMetaData().getDatabaseProductName().contains("Informix"));
                 }
                 catch (Throwable ex) {
-//                    logger.debug("Could not check database product name", ex);
+                    Scope.getCurrentScope().getLog(StatementCreatorUtils.class).fine("Could not check database product name", ex);
```
Comment (MalloD12) [inline liquibase-standard/src/main/java/liquibase/executor/jvm/StatementCreatorUtils.java:85]: Something very minor, but I would keep the same log level (debug) here or at least change it as a `warning`.

**Comment by MalloD12** [comment]
Hhi @asolntsev,

Thank you for submitting another PR for Liquibase. In general, this PR looks good to me. The only thing I have for commenting here is in some cases we are logging an error message in the `catch` block as a `warning` or even maybe as `info` log message. I think maybe we can update these to the right log level. What do you think about it?

Thanks,
Daniel.

File: liquibase-standard/src/main/java/liquibase/executor/jvm/StatementCreatorUtils.java (hunk 1 above)
Comment (asolntsev) [reply liquibase-standard/src/main/java/liquibase/executor/jvm/StatementCreatorUtils.java:85]: @MalloD12 But "fine" is the same as "debug". :)

Look, this is javadoc of `liquibase.logging.Logger`:

image

**Comment by asolntsev** [comment]
> I think maybe we can update these to the right log level

Generally I like the idea :)

But I intentionally didn't change any logic to keep the PR scope as small as possible. 
In some cases, "info" might be totally fine:
1. e.g. if LiquiBase could not get DB version because DB connection has been closed by this moment
2. or if DB doesn't support such a function.

File: liquibase-standard/src/main/java/liquibase/executor/jvm/StatementCreatorUtils.java (hunk 1 above)
Comment (MalloD12) [reply liquibase-standard/src/main/java/liquibase/executor/jvm/StatementCreatorUtils.java:85]: Yeah, according to that class, it's correct what you have done. But I'm not sure why `debug` methods are deprecated here. I'll check with the team. 

Thanks,
Daniel.

**Review by MalloD12 — APPROVED** [review]
**Approved**.

PR looks good , I had only a doubt regarding some log level messages but after reviewing with the team we are ok as it is. 

Thank you, @asolntsev for one other PR.

**Review by rberezen — APPROVED** [review]

File: liquibase-standard/src/main/java/liquibase/datatype/core/UUIDType.java [hunk 2]
```diff
@@ -42,11 +42,7 @@ public DatabaseDataType toDatabaseDataType(Database database) {
                 return new DatabaseDataType("UUID");
             }
         } catch (DatabaseException e) {
-            try {
```
Comment (filipelautert) [inline liquibase-standard/src/main/java/liquibase/datatype/core/UUIDType.java:45]: lol this one was fun

**Review by filipelautert — APPROVED** [review]

**Pull request merged**